install.rst

```diff
@@ -18,49 +18,135 @@
 Installing Zeek
 ===============
 
-Prerequisites
+To run Zeek, grab our official Docker images, download our Linux binary
+packages, install via Homebrew on your Mac, use the ports collection on FreeBSD,
+or build Zeek yourself.
+
+Docker Images
 =============
 
-Before installing Zeek, you'll need to ensure that some dependencies
-are in place.
+We provide official Docker images on Docker Hub at https://hub.docker.com/u/zeekurity.
```

Comment:
Just wondering, I thought there was some work on the way to publish images to the ECR as well (https://gallery.ecr.aws/?searchTerm=zeek), but I cannot find the ticket.

Reply:
Yes that's stalled on me, I've been meaning to explore CLI access to the AWS ECR to sync over the existing images. I don't think there's currently a ticket.

Reply:
Not sure about CLI, but publishing from CI is pretty simple with the actions we already use (just requires an additional step to log in at ECR, and another remote to push to); the Spicy setup which already does it is [here](https://github.com/zeek/spicy/blob/main/.github/workflows/docker.yml).